docs/Reference/JSON-RPC-API-Methods.md

```diff
@@ -2191,4 +2191,193 @@ Proposes [adding or removing a validator](../Consensus-Protocols/IBFT.md#adding-
 
 ## Permissioning Methods
 
-Permissioning is under development and will be available in v1.0. 
\ No newline at end of file
+!!! note 
+    Permissioning is under development and will be available in v1.0. 
+
+!!! note
+    The `PERM` API methods are not enabled by default. Use the [`--rpc-http-api`](Pantheon-CLI-Syntax.md#rpc-http-api) 
+    or [`--rpc-ws-api`](Pantheon-CLI-Syntax.md#rpc-ws-api) options to enable the `PERM` API methods.
+
+### perm_addAccountsToWhitelist
+
+Adds accounts (participants) to the accounts whitelist. 
+
+**Parameters** 
+
+`list of strings` - List of account addresses 
+
+**Returns** 
+
+`result` - `Success` or `error`
+
+!!! example
+    ```bash tab="curl HTTP request"
+    $ curl -X POST --data '{"jsonrpc":"2.0","method":"perm_addAccountsToWhitelist","params":[["0xb9b81ee349c3807e46bc71aa2632203c5b462032", "0xb9b81ee349c3807e46bc71aa2632203c5b462034"]], "id":1}' <JSON-RPC-http-endpoint:port>
+    ```
+    
+    ```bash tab="wscat WS request"
+    {"jsonrpc":"2.0","method":"perm_addAccountsToWhitelist","params":[["0xb9b81ee349c3807e46bc71aa2632203c5b462032", "0xb9b81ee349c3807e46bc71aa2632203c5b462034"]], "id":1}
+    ```
+    
+    ```json tab="JSON result"
+    {
+        "jsonrpc": "2.0",
+        "id": 1,
+        "result": "Success"
+    }
+    ```
+    
+### perm_getAccountsWhitelist
+
+Lists accounts (participants) in the accounts whitelist. 
+
+**Parameters** 
+
+None
+
+**Returns** 
+
+`result: list` - Accounts (participants) in the accounts whitelist. 
+
+!!! example
+    ```bash tab="curl HTTP request"
+    $ curl -X POST --data '{"jsonrpc":"2.0","method":"perm_getAccountsWhitelist","params":[], "id":1}' <JSON-RPC-http-endpoint:port>
+    ```
+    
+    ```bash tab="wscat WS request"
+    {"jsonrpc":"2.0","method":"perm_getAccountsWhitelist","params":[], "id":1}
+    ```
+    
+    ```json tab="JSON result"
+    {
+        "jsonrpc": "2.0",
+        "id": 1,
+        "result": [
+            "0x0000000000000000000000000000000000000009",
+            "0xb9b81ee349c3807e46bc71aa2632203c5b462033"
+        ]
+    }
+    ``` 
+    
+### perm_removeAccountsFromWhitelist
+
+Removes accounts (participants) from the accounts whitelist. 
+
+**Parameters** 
+
+`list of strings` - List of account addresses 
+
+**Returns** 
+
+`result` - `Success` or `error`
```

Comment:
do we explain when you would expect an error?

Reply:
Added